modules/_node-scripts/util/forkModule.mjs

```diff
@@ -0,0 +1,45 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import childProcess from 'child_process';
+
+import onExit from './onExit.mjs';
+
+export default async function forkModule(modulePath, params, options) {
```

Comment:
Not sure I fully understand the point of this, what do we use this for?

Reply:
This is for running `npm-scripts` and `tsc` but I may swith to `execa`, as you suggested, if it can be used here.

Reply:
Will be addressed in https://liferay.atlassian.net/browse/LPD-27158